das/daser.go

```diff
@@ -236,4 +280,10 @@ func (d *DASer) catchUp(ctx context.Context, job *catchUpJob) {
 
 	log.Infow("successfully caught up", "from", job.from,
 		"to", job.to, "finished (s)", time.Since(routineStartTime))
+	// report successful result
+	return &catchUpResult{
+		id:         job.id,
+		checkpoint: job.to,
```

Comment:
Ids are fine. Also, note that you could do the same behavior by using checkpoints(job.to) only, because it also unique incremental value, like the id

Reply:
i'm changing it to `job.to` to simplify it.